pyproject.toml

```diff
@@ -25,6 +25,7 @@ python-dotenv = "^1.0.0"
 rich = "^13.5.2"
 tabulate = "*"
 pick = "^2.2.0"
+pyperclip = "^1.8.2"
```

Comment:
pyperclip seems abandoned: https://github.com/asweigart/pyperclip/issues/250 (not an endorsement of the alternative in the issue)

pyperclip also has `pytest` in their non-dev deps, which I found strange and undesireable.